projects/plugins/jetpack/extensions/blocks/payments-intro/edit.js

```diff
@@ -1,15 +1,37 @@
-import {
-	InnerBlocks,
-	__experimentalBlockVariationPicker as BlockVariationPicker, // eslint-disable-line wpcalypso/no-unsafe-wp-apis
-	store as blockEditorStore,
-} from '@wordpress/block-editor';
+import { InnerBlocks, store as blockEditorStore } from '@wordpress/block-editor';
 import { createBlock, getBlockType, registerBlockVariation } from '@wordpress/blocks';
+import { Button, Placeholder } from '@wordpress/components';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import { get } from 'lodash';
 import defaultVariations from './variations';
 
+const JetpackBlockVariationPicker = function ( { variations, onSelect } ) {
```

Comment:
The name suggests that this is a shared component but then it's located in the payments-intro directory. Maybe we could move this to: projects/plugins/jetpack/extensions/shared/components/jetpack-variation-picker?

Reply:
I agree, at first, I wasn't even going to comment because of it only being used once XD. 

I do think that we could move it to a subdir the only risk I see is that if a different team besides us need to use the component they probably would create a new one (re-inventing the wheel) since the likelihood of a random team knowing about the existence of the JetpackBlockVariationPicker if it is in a Block's sub-directory is low.

Having said this, this isn't a blocker so feel free to create a new issue, implement any changes here, or discard this conversation.

Reply:
I moved it to a subdir and renamed.